locksmith/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const MAX_UINT =
```

Comment:
Do we not have this already? If not please use `ethers` which already defines it.

Reply:
not present in `locksmith`, replacing with `ethers`